Sources/SwiftlyCore/HTTPClient.swift

```diff
@@ -115,8 +304,19 @@ public struct SwiftlyHTTPClient {
             return snapshot
         }
 
-        return try await self.mapGitHubTags(limit: limit, filterMap: filter) { page in
-            try await self.getTags(page: page)
+        snapshotToolchains.formUnion(Set(swiftOrgFiltered))
+
+        guard !swiftOrgFiltered.isEmpty else {
+            return []
+        }
+
+        var finalSnapshotToolchains = Array(snapshotToolchains)
+        finalSnapshotToolchains.sort(by: >)
+
+        return if let limit = limit {
+            Array(finalSnapshotToolchains[0..<limit])
+        } else {
+            finalSnapshotToolchains
```

Comment:
We have several variables for the snapshots, but I think we only really need `swiftOrgFiltered`. Can we drop `finalSnapshotToolchains` and `snapshotToolchains`?

Reply:
I'm looking into making this simpler and easier to follow.